StyleCop.Analyzers/StyleCop.Analyzers.Test.CSharp10/ReadabilityRules/SA1141CSharp10UnitTests.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Tunnel Vision Laboratories, LLC. All Rights Reserved.
```

Comment:
Good catch with these missing test classes. I assume it was a mistake to not add them earlier. Would be good to have automated checks for things like this.